pyorbital/orbital.py

```diff
@@ -133,21 +133,43 @@ def get_observer_look(sat_lon, sat_lat, sat_alt, utc_time, lon, lat, alt):
     else:
         az_data = az_
 
+    # When elevation is 90 degrees both top_s and top_e are close to zero
+    # and azimuth angle is undefined.
+    undet_azi = np.logical_and(top_s == 0, top_e == 0)
     if has_dask and isinstance(az_data, da.Array):
+        az_data = da.where(undet_azi, 0, az_data)
         az_data = da.where(top_s > 0, az_data + np.pi, az_data)
         az_data = da.where(az_data < 0, az_data + 2 * np.pi, az_data)
     else:
+        az_data[np.where(undet_azi)] = 0
         az_data[np.where(top_s > 0)] += np.pi
         az_data[np.where(az_data < 0)] += 2 * np.pi
 
+    rg_ = np.sqrt(rx * rx + ry * ry + rz * rz)
+
+    top_z_divided_by_rg_ = top_z / rg_
+    el_ = np.arcsin(top_z_divided_by_rg_)
```

Comment:
Using `clip` might help avoid the extra filtering later:
```suggestion
    top_z_divided_by_rg_ = top_z / rg_
    top_z_divided_by_rg_ = top_z_divided_by_rg_.clip(max=1)
    el_ = np.arcsin(top_z_divided_by_rg_)
```